src/lib/assetBridger/ethBridger.ts

```diff
@@ -141,6 +142,36 @@ export class EthBridger extends AssetBridger<
     return new EthBridger(await getL2Network(l2Provider))
   }
 
+  /**
+   * Returns the addresses of all contracts that make up the ETH bridge
+   * @param rollupContractAddress Address of the Rollup contract
+   * @param l1Provider An L1 provider
+   * @returns EthBridge object with all information about the ETH bridge
+   */
+  public async getBridgeInformation(
+    rollupContractAddress: string,
+    l1Provider: Provider
+  ): Promise<EthBridge> {
+    await this.checkL1Network(l1Provider)
+
+    const rollup = RollupAdminLogic__factory.connect(
+      rollupContractAddress,
+      l1Provider
+    )
```

Comment:
you don't need this, there are `getL1Network` and `getL2Network` that return the same thing
https://github.com/OffchainLabs/arbitrum-sdk/blob/5f622e10f5cb871edb90c3c2bb6e39b1e2d0cf0f/src/lib/dataEntities/networks.ts#L274-L283